docs/configuration-reference/components/contour.md

```diff
@@ -22,9 +22,7 @@ Currently the supported platform is Packet.
 
 ## Prerequisites
 
-* A Lokomotive cluster accessible via `kubectl` deployed on Packet.
-
-* [MetalLB component](metallb.md) installed and configured.
```

Comment:
I don't think this change is right.

This document says "Currently the supported platform is Packet", but it is now supported on AWS too as far as I understand. So I'd remove the sentence "Currently the supported platform is Packet" too.

If users want to run contour on Packet they do need metallb, but this is mentioned in the ingress with Contour and Metallb How To so that's OK.

As a general comment, it would be helpful to know the reasoning of changes in commit messages.

Reply:
On it